doc/source/whatsnew/v0.23.1.txt

```diff
@@ -81,7 +81,7 @@ Indexing
 I/O
 ^^^
 
--
+- bug when :meth:`json_normalize` was called with `None` values in nested levels in JSON  (:issue:`21158`)
```

Comment:
Capitalize "Bug" and as Jeff mentioned use double backticks around None, not just single